Pod/Classes/JSSAlertView.swift

```diff
@@ -181,19 +181,19 @@ public class JSSAlertView: UIViewController {
 		let titleString = titleLabel.text! as NSString
 		let titleAttr = [NSFontAttributeName:titleLabel.font]
 		let titleSize = CGSize(width: contentWidth, height: 90)
-		let titleRect = titleString.boundingRectWithSize(titleSize, options: NSStringDrawingOptions.UsesLineFragmentOrigin, attributes: titleAttr, context: nil)
+		let titleRect = titleString.boundingRect(with: titleSize, options: .usesLineFragmentOrigin, attributes: titleAttr, context: nil)
 		yPos += padding
-		self.titleLabel.frame = CGRect(x: self.padding, y: yPos, width: self.alertWidth - (self.padding*2), height: ceil(titleRect.size.height))
-		yPos += ceil(titleRect.size.height)
+		self.titleLabel.frame = CGRect(x: self.padding, y: yPos, width: self.alertWidth - (self.padding*2), height: ceil(titleRect.height))
+		yPos += ceil(titleRect.height)
 		
 		
 		// position text
 		if self.textView != nil {
 			let textString = textView.text! as NSString
 			let textAttr = [NSFontAttributeName:textView.font as! AnyObject]
```

Comment:
And this: let textAttr = [NSFontAttributeName: textView.font!] as [String: Any]?